bubblewrap.c

```diff
@@ -1861,8 +1872,14 @@ parse_args_recurse (int          *argcp,
         {
           die ("Unknown option %s", arg);
         }
+      /* implicit argv0 to be execvp'ed */
       else
         {
+          /* check if the --argv0 was informed too */
+          if (opt_argv0)
```

Comment:
I'm confused...it seems like this will entirely break option processing if `--argv0` is specified?